tests/parm/input.benchmark_ccpp.nml.IN

```diff
@@ -0,0 +1,309 @@
+
```

Comment:
Pease remove the entire file input.benchmark_ccpp.nml.IN

Reply:
The file input.benchmark_ccpp.nml.IN' was removed.